plugins/snapshots/erofs/erofs_linux.go

```diff
@@ -507,6 +535,8 @@ func (s *snapshotter) Remove(ctx context.Context, key string) (err error) {
 		if err != nil {
 			return fmt.Errorf("unable to get directories for removal: %w", err)
 		}
+		// Clear IMMUTABLE_FL before removal, since this flag avoids it.
+		_ = setImmutable(s.layerBlobPath(id), false)
```

Comment:
I think we should return this error here.

Reply:
Yeah, will fix later.

I once thought administers manually house-keeping or unsupported fs use cases or lack of CAP_LINUX_IMMUTABLE due to MAC restriction.

But since setImmutable won't SETFLAGS if (GETFLAG & FS_IMMUTABLE_FL == 0), I'm fine with it now.

Reply:
Fixed, thanks for your comment!